index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```

Comment:
id attributes can be added to the headings, making it easier to reference these elements programmatically in JavaScript if needed. (game-over) and (game-over-refresh)